Work around compiler bug in older GCC

Oder GCCs don't like the scope resolution operator in the contest below. Replacing this with `namespace` keywords should fix it. I'm working on replicating this on falcon (all my machines and VMs have gcc versions that are too new and do not have this bug).

Tagging @amanichen 

```
/home/a.cheniour/projects/magpie/build/header_symlinks/RadialGreensConvolution.h:172:26: error: specialization of ‘template<class T> class libMesh::Parallel::StandardType’ in different namespace [-fpermissive]
 class libMesh::Parallel::StandardType<RadialGreensConvolution::QPData> : public DataType
```

--- a/include/userobjects/RadialGreensConvolution.h
+++ b/include/userobjects/RadialGreensConvolution.h
@@ -168,13 +168,20 @@ template <>
 const Point & PointListAdaptor<RadialGreensConvolution::QPData>::getPoint(
     const RadialGreensConvolution::QPData & item) const;
 
+namespace libMesh
+{
+namespace Parallel
+{
+
 template <>
-class libMesh::Parallel::StandardType<RadialGreensConvolution::QPData> : public DataType
+class StandardType<RadialGreensConvolution::QPData> : public DataType
 {
 public:
   explicit StandardType(const RadialGreensConvolution::QPData * example = nullptr);
   StandardType(const StandardType<RadialGreensConvolution::QPData> & t);
   ~StandardType() { this->free(); }
 };
+}
+}
 
 #endif // RADIALGREENSCONVOLUTION_H

--- a/src/userobjects/RadialGreensConvolution.C
+++ b/src/userobjects/RadialGreensConvolution.C
@@ -541,7 +541,12 @@ RadialGreensConvolution::updateCommunicationLists()
   _update_communication_lists = false;
 }
 
-libMesh::Parallel::StandardType<RadialGreensConvolution::QPData>::StandardType(
+namespace libMesh
+{
+namespace Parallel
+{
+
+StandardType<RadialGreensConvolution::QPData>::StandardType(
     const RadialGreensConvolution::QPData * example)
 {
   // We need an example for MPI_Address to use
@@ -587,6 +592,8 @@ libMesh::Parallel::StandardType<RadialGreensConvolution::QPData>::StandardType(
   this->commit();
 
 #endif // LIBMESH_HAVE_MPI
+}
+}
 }
 
 StandardType<RadialGreensConvolution::QPData>::StandardType(

